app/gui/view/documentation/src/lib.rs

```diff
@@ -236,7 +223,12 @@ ensogl::define_endpoints! {
     Input {
         /// Display documentation of the specific entry from the suggestion database.
         display_documentation (EntryDocumentation),
-        show_hovered_item_preview_caption(bool),
+        /// Open documentation panel with animation.
+        show(),
+        /// Close documentation panel with animation.
+        hide(),
```

Comment:
How about using `set_visible (bool)` instead of `show/hide`? It looks like it would be both easier to use (since it is converted from bool at usage point anyway) and result in less FRP nodes created.